### PR TITLE
NotebookWidgetFactory tests

### PR DESCRIPTION
**Pull request by blink1073**

### CODE REVIEW

File: src/notebook/notebook/widgetfactory.ts [hunk 1]
```diff
@@ -41,14 +41,21 @@ export
...
   /**
    * Construct a new notebook widget factory.
+   *
+   * @param rendermime - The rendermime instance.
+   *
+   * @param clipboard - The application clipboard.
    */
   constructor(rendermime: RenderMime<Widget>, clipboard: IClipboard) {
-    this._rendermime = rendermime.clone();
+    this._rendermime = rendermime;
```
Comment (jasongrout) [inline src/notebook/notebook/widgetfactory.ts]: Good job catching this extra `.clone()`!

**Comment by jasongrout** [comment]
Code looks great, tests pass.

**Comment by jasongrout** [comment]
Ah, except it apparently needs a rebase now...

**Comment by blink1073** [comment]
Rebased.

**Pull request merged**